build/media_source/plg_system_shortcut/js/keyselect.es6.js

```diff
@@ -9,23 +9,31 @@
 
   const keyDown = (e) => {
     if (e.keyCode >= 65 && e.keyCode <= 90) {
-      e.preventDefault();
-      e.stopPropagation();
-      e.stopImmediatePropagation();
-    }
-  };
-
-  const keyUp = (e) => {
-    if (e.keyCode >= 65 && e.keyCode <= 90) {
+      const isMac = /(Mac|iPhone|iPod|iPad)/i.test(navigator.platform);
       const pressedKeys = [];
+      if (e.metaKey) {
+        if (isMac) {
```

Comment:
Could replace the if/else everywhere with a function call that returns the equivalent for the platform.